api/docs/api.md

```diff
@@ -139,6 +139,7 @@ Docs: https://v3-apidocs.cloudfoundry.org/version/3.100.0/index.html#processes
 | Get Process | GET /v3/processes/\<guid>/sidecars |
 | Get Process Sidecars | GET /v3/processes/\<guid>/sidecars |
 | Scale Process | POST /v3/processes/\<guid>/actions/scale |
+| Get Process Stats | POST /v3/processes/\<guid>/stats |
```

Comment:
It might be worth documenting what parts of this we do/don't support since I imagine we'll expand it over time and it will be helpful to know what the current state is.

Reply:
Addressed in commit https://github.com/cloudfoundry/cf-k8s-controllers/pull/248/commits/5a4d14ec897981ac8a15f7962ccf70c663466818